requirements-dev.txt

```diff
@@ -15,6 +15,7 @@ clang-format~=16.0
 isort==5.13.2
 click==8.0.4
 cmake
+jax[cpu]==0.4.28
```

Comment:
Do we need to add this also on `requirements.txt`?

Reply:
So far we don't, as we don't need it for building Lightning.
This is more of a test constraint.